src/main/java/duke/command/CommandMarkDone.java

```diff
@@ -32,24 +32,24 @@ public void execute(TaskList taskList) {
      * @param task The Task that has been completed
      * @param taskList The TaskList containing all tasks.
      */
-    private void releaseQueuedTasks(Task task, TaskList taskList) {
-        if (task.isQueuedTasks()) {
-            generateQueuedTasks(task, taskList);
-        }
-    }
-
-    /**
-     * Transfers all queued tasks from the trigger Task to the TaskList.
-     * @param task The triggering Task
-     * @param taskList The TaskList containing all tasks
-     */
-    private void generateQueuedTasks(Task task, TaskList taskList) {
-        TaskList newTasks = task.getQueuedTasks();
-        for (Task newTask : newTasks.getList()) {
-            taskList.addTask(newTask);
-        }
-        task.setQueuedTasks(null);
-    }
+//    private void releaseQueuedTasks(Task task, TaskList taskList) {
+//        if (task.isQueuedTasks()) {
+//            generateQueuedTasks(task, taskList);
+//        }
+//    }
+//
+//    /**
+//     * Transfers all queued tasks from the trigger Task to the TaskList.
+//     * @param task The triggering Task
+//     * @param taskList The TaskList containing all tasks
+//     */
+//    private void generateQueuedTasks(Task task, TaskList taskList) {
+//        TaskList newTasks = task.getQueuedTasks();
+//        for (Task newTask : newTasks.getList()) {
+//            taskList.addTask(newTask);
+//        }
+//        task.setQueuedTasks(null);
+//    }
```

Comment:
Good job catching code that was not used. Yep, I initially intended to abstract the code but realized it's kind of meaningless to do so so I merged it as seen in the method 'loadQueuedTasks' but I forgot to delete the abstracted code. Instead of commenting it out, could you simply delete it entirely, along with the documentation? Commented out code decreases our code quality. Thanks!